ui/src/app/kvm/components/VfResources/VfResources.tsx

```diff
@@ -18,21 +18,30 @@ const VfResources = ({
 }: Props): JSX.Element => {
   let content: ReactNode;
   if (showAggregated) {
-    const [allocatedVFs, freeVFs] = interfaces.reduce<[number, number]>(
-      ([allocated, free], { virtual_functions }) => {
+    const [allocatedVFs, freeVFs, otherVFs] = interfaces.reduce<
+      [number, number, number]
```

Comment:
It might be slightly safer to define the types as the same as those from `PodResource` even though they're not strictly the same type (because they're the total of the values).
```suggestion
      [PodResource["allocated_tracked"], PodResource["free"], PodResource["allocated_other"]]
```

Reply:
Good call